Seneca-CDOT/telescope
# Responsive SVG art, and removed tutorial

## Issue This PR Addresses  
Fixes #3545, fixes #3544

## Type of Change




## Description
Made the SVG art responsive as requested in #3545 and removed the tutorial panel as requested in #3544

## Steps to test the PR
> * `cd src/web/docusaurus`
> * Might need to also do `pnpm install`
> * `pnpm start`
> * Your browser should automatically open `http://localhost:4631`

Testing steps from #3544 

## Screenshots
![desktop screen](https://i.imgur.com/N79OpCD.png)
![mobile screen](https://i.imgur.com/l3AGPe5.png)

## Checklist
